hdfs3/core.py

```diff
@@ -245,14 +273,14 @@ def chown(self, path, owner, group):
         return out == 0
 
     def cat(self, path):
-        "Return contents of file"
-        buff = b''
+        """ Return contents of file """
+        buffers = []
         with self.open(path, 'r') as f:
             out = 1
             while out:
                 out = f.read(2**16)
-                buff = buff + out
```

Comment:
Performance note:  I believe that Python strings/bytes are immutable, so this line caused a lot of heavy copies.